index.html

```diff
@@ -1,28 +1,101 @@
+                <th class="td-b" colspan="4"><span class="filter-label">buffers</span></td>
+            </tr>
+            <tr id="clearAll">
+                <td class="td-r"><div id="clearAll-r"><p class="label-x">all</p></div></td>
+                <td class="td-b" colspan="4">&nbsp;</td>
+            </tr>
+
+            <!-- ======= regions ======= -->
+            <tr id="AL">
+                <td class="td-r"><div class="region"><p class="label-r">AL</p></div></td>
+                <td class="td-b"><div class="buffer ft500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft1000">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft2500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft5280">&nbsp;</div></td>
+            </tr>
+            <tr id="AR">
+                <td class="td-r"><div class="region"><p class="label-r">AR</p></div></td>
+                <td class="td-b"><div class="buffer ft500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft1000">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft2500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft5280">&nbsp;</div></td>
+            </tr>
+            <tr id="DC">
+                <td class="td-r"><div class="region"><p class="label-r">DC</p></div></td>
+                <td class="td-b"><div class="buffer ft500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft1000">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft2500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft5280">&nbsp;</div></td>
+            </tr>
+            <tr id="MO">
+                <td class="td-r"><div class="region"><p class="label-r">MO</p></div></td>
+                <td class="td-b"><div class="buffer ft500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft1000">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft2500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft5280">&nbsp;</div></td>
+            </tr>
+            <tr id="PG">
+                <td class="td-r"><div class="region"><p class="label-r">PG</p></div></td>
+                <td class="td-b"><div class="buffer ft500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft1000">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft2500">&nbsp;</div></td>
+                <td class="td-b"><div class="buffer ft5280">&nbsp;</div></td>
+            </tr>
+        </table>
+        <div class="locText start">&nbsp;</div><span class="filter-label">start</span>
+        <div class="locText end">&nbsp;</div><span class="filter-label">end</span>
+    </div>
+
+    <!-- ======= map ======= -->
+    <div id="map"></div>
+
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
Any reason jQuery 1.11.3 is used instead of the more recent 1.12.4?

Also, the jQuery uses SubResource Integrity (SRI) Checking for extra security, I'd prefer to use that code if possible (from https://code.jquery.com/):

```
<script src="https://code.jquery.com/jquery-1.12.4.js" integrity="sha256-Qw82+bXyGq6MydymqBxNPYTaUXXq7c8v3CwiYwLLNXU=" crossorigin="anonymous"></script>
```